script.js

```diff
@@ -0,0 +1,41 @@
+
+var userName = prompt("Hi there. What is your name?");
+
+if (userName!= null) {
+   alert("Welcome " + userName + ".  My name is Charlie.  I am a program that will assess potential personality traits and how you may learn best.  Let's get started. ");
+}
+
+//First Selection (Choice of Profession)
+
+var jobChoices = ["lawyer" , "president" , "garbage man" , "chef", "entrepreneur"];
+var userJob =  prompt("What do you want to be? A " + jobChoices );
+
+if (userJob === jobChoices[0]) {
+ alert("You are a negotiator.")}
+else if (userJob === jobChoices[1]) {
+ alert("You are confident.")}
+else if (userJob === jobChoices[2]) {
+ alert("You are humble.")}
+else if (userJob === jobChoices[3]) {
+ alert("You are creative.")}
+else if (userJob === jobChoices[4]) {
+ alert("You are a risk-taker.")}
+else {for (var i = 0; i < 100; i++) {
+// console.log("Uh oh, next time.  You will spell right.")
+alert("Your fired. You need to spell better.[i]")}
+};
```

Comment:
I know, I still need to work on... But also need to get started on project... Need to weigh options...

Tried to make some sort of like learning game... Wrong idea

Get Outlook for iOS<https://aka.ms/o0ukef>




On Fri, Jul 8, 2016 at 11:17 AM -0400, "James Reichard" <notifications@github.com<mailto:notifications@github.com>> wrote:


In script.js<https://github.com/ga-wdi-exercises/choose_your_own_adventure_js/pull/197#discussion_r70090658>:

> +var userJob =  prompt("What do you want to be? A " + jobChoices );
> +
> +if (userJob === jobChoices[0]) {
> + alert("You are a negotiator.")}
> +else if (userJob === jobChoices[1]) {
> + alert("You are confident.")}
> +else if (userJob === jobChoices[2]) {
> + alert("You are humble.")}
> +else if (userJob === jobChoices[3]) {
> + alert("You are creative.")}
> +else if (userJob === jobChoices[4]) {
> + alert("You are a risk-taker.")}
> +else {for (var i = 0; i < 100; i++) {
> +// console.log("Uh oh, next time.  You will spell right.")
> +alert("Your fired. You need to spell better.[i]")}
> +};


evil

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ga-wdi-exercises/choose_your_own_adventure_js/pull/197/files/d8fbfd3870a8e00c397bdeab3e646561ad3d1288#r70090658>, or mute the thread<https://github.com/notifications/unsubscribe/ATF8hO6-wyzIjTenkxQX2QVIbVxhzvmzks5qTmn2gaJpZM4JEth7>.